lib/rubocop/options.rb

```diff
@@ -615,7 +615,9 @@ module OptionsHelp
       version:                          'Display version.',
       verbose_version:                  'Display verbose version.',
       parallel:                         ['Use available CPUs to execute inspection in',
-                                         'parallel. Default is true.'],
+                                         'parallel. Default is true.',
+                                         'You can specify the number of parallel processes',
```

Comment:
Same. I use "using" being used in the longer text. :-) Maybe that's what you meant to put here as well.